src/threads.h

```diff
@@ -276,15 +276,19 @@ enum {
     pthread_set_name_np(pthread_self(), tname); \
     0; \
 })
+#define SCGetThreadName(tname, tnlen) ({ pthread_get_name_np(pthrad_self(), tname, tnlen); })
```

Comment:
Pretty sure this should be `pthread_self`..  Was trying to confirm this worked OK on FreeBSD but am hitting other build issues at the moment.

Reply:
On FreeBSD I get this:
```
--- util-debug.o ---
util-debug.c:357:48: error: invalid operands to binary expression ('void' and 'int')
        if (SCGetThreadName(_sc_subsystem, 16) < 0) {
```
From the FreeBSD man page, `pthread_get_name_np` returns void